### DISCUSSION TITLE
Potential fix against community tournaments being severely understaffed

### DISCUSSION THREAD

**Discussion by DerHamm3r** [ppy/osu#25599] [Ideas]
Hey, I am putting this here as a follow-up to a discussion we had a week ago on the #tournaments channel on the dev discord.

I have been staffing in community tournaments for almost 2 years now, and as of recent it felt like the amount of people volunteering to staff in those has been on the decline, while at the same time the number of people playing in them being roughly the same. And from what other hosts and staff told me, this seems to be a general feeling for most of them. I have also seen a lot of ideas previously discussed on the github which would lead to some big changes. I am sure that Lazer will implement a lot of cool features which will take off the workload from staff (like using something like the playlists as a form of qualifier, removing the need for referees for one stage), but this suggestion is more of a simple and immediate fix. And it comes in the form of **exposure** for what staffing is, and easier access to information needed to get into it.

I've always found it a bit sad that community tournaments only have a forum for players to find and sign up for them, with some forum posts having a small staff signup link. And you kinda have to know that it is there. If you go on the website and click on community -> tournaments all you see are the official osu world cups. So my Idea is the following: Rename the "tournaments" option to "official tournaments" and add a second page for "community tournaments". It should start off with a section for players, shortly telling them what a community tournament is, how easy it is to sign up, maybe a short 3-5 minute video explaining it, and then a link to the forum where people can find up tournaments to sign up to. 

Then below that would be a section for staff. A small section about what is required to run a tournament, required positions, a hint that new staff is always needed, and some resources that help people to get into it. Some positions, especially referees, are always in need and learning to ref is not hard. So it should be something that sounds like it would be a fun thing to get into.

Another thing to make it easier to get into staffing would be some sort of "mentorship program". Me and some other hosts that I talked to would be really interested in being available for new staff and to teach them all they need to know to get started. Getting into staffing can be a bit scary since people will get anxious of messing up and ruining a tournament. So having people willing to take them by the hand and showing them things could be motivation for some to get into staffing. 

The last suggestion would be the one thing that would take the most effort to implement. Reward staff. This can only really be tracked in badged community tournaments, but modern sheets track who did what, so adding an extra step when applying for a badge at the end of a tournament by having hosts submit a list of work done by people. Then, every time someone reffed, streamed, commentated X amount of matches, or spend X hours pooling / playtesting they could be rewarded with a month of supporter for example, or a smaller version of the owc badges, or a title like "community tournament staff" which lasts for 3 months after they completed their last tournament, motivating them to keep going to keep their title. I know that this is not an easy one to do since there is a lot of room to abuse this, and I doubt that this will happen, but I think it is still worth mentioning and explaining.

A smaller version of the suggestion above would be to give streamers and commentators the ability to submit a request to remove the need of supporter for them to use the spectator client or lazer. I personally barely play osu anymore and am only paying for supporter to stream tournaments right now. 

I am looking forward to all the big and new tournament related changes that we will get with Lazer in the future, but these ideas would be a temporary fix for what we have at the moment. And outside of my last suggestion it would be a smaller website related change which shouldn't be too much work.

**Comment by peppy** [comment]
You call the changes a "temporary fix" but implementing what you want would require hundreds of hours of time over a year, including upkeep.

Might be best to look for solutions elsewhere. osu! is a community and historically has leaned on the community to make these things happen as much as possible, not me.

We do intend to give the tournament system a revamp at some point but i doubt this will change your "staffing" issue. Maybe as more tournament tools come online in lazer you'll need less "staff" though.
